docker-compose.override.yaml

```diff
@@ -7,9 +7,19 @@ services:
       target: development
     image: ffc-pay-file-receiver-development
     container_name: ffc-pay-file-receiver-development
-    ports:
-      - "3000:3000"
-      - "9229:9229"
```

Comment:
Should expose `9229` bound to "something" as that is the Node.js debug port.  Without that exposed it wouldn't be possible to attach a debugger when running the app in a container.